src/handlers/gestures/GestureDetector/Wrap.web.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef } from 'react';
+import type { LegacyRef, PropsWithChildren } from 'react';
+import { tagMessage } from '../../../utils';
+
+let wrapID = 0;
```

Comment:
Do we need the ids?

Reply:
Seems like we don't. I've used that to check if view was added by Gesture Handler, but it looks like checking for `display: contents;` is enough

Removed in [7a11ad5](https://github.com/software-mansion/react-native-gesture-handler/pull/3127/commits/7a11ad5b0bb6c24761e34447b0e91b2e1106b3c3)